<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kkshell</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/application.png</normaloff>:/images/application.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <widget class="QMenu" name="menunew">
     <property name="title">
      <string> new session</string>
     </property>
     <addaction name="actionNewSerialSession"/>
     <addaction name="actionNewSSHSession"/>
     <addaction name="actionNewLocalShellSession"/>
    </widget>
    <addaction name="menunew"/>
    <addaction name="actionOpenSession"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuedit">
    <property name="title">
     <string>edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionPaste"/>
    <addaction name="actionFind"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menutools">
    <property name="title">
     <string>tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>about</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>view</string>
    </property>
    <addaction name="actionShowToolsBar"/>
    <addaction name="actionShowButtonBar"/>
    <addaction name="actionShowStatusBar"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuedit"/>
   <addaction name="menuview"/>
   <addaction name="menutools"/>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolsBar">
   <property name="windowTitle">
    <string>tools bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenSession"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
   <addaction name="actionClear"/>
  </widget>
  <widget class="QToolBar" name="commandButtonBar">
   <property name="windowTitle">
    <string>command button bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>disconnect</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/setting.png</normaloff>:/images/setting.png</iconset>
   </property>
   <property name="text">
    <string>settings</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>copy selected</string>
   </property>
   <property name="toolTip">
    <string>copy selected</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+V</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
   </property>
   <property name="text">
    <string>find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionOpenSession">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>open session</string>
   </property>
  </action>
  <action name="actionNewSerialSession">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>new serial session</string>
   </property>
  </action>
  <action name="actionNewSSHSession">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>new ssh session</string>
   </property>
  </action>
  <action name="actionNewLocalShellSession">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>new local shell session</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>copy all</string>
   </property>
   <property name="toolTip">
    <string>copy all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionShowToolsBar">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/show.png</normaloff>:/images/show.png</iconset>
   </property>
   <property name="text">
    <string>tools bar</string>
   </property>
  </action>
  <action name="actionShowButtonBar">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/show.png</normaloff>:/images/show.png</iconset>
   </property>
   <property name="text">
    <string>button bar</string>
   </property>
  </action>
  <action name="actionShowStatusBar">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/show.png</normaloff>:/images/show.png</iconset>
   </property>
   <property name="text">
    <string>status bar</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
